### ISSUE TITLE
Deconstructed params getting swapped in 0.3.10.

### ISSUE THREAD

**Issue by creack** [hosseinmd/prettier-plugin-jsdoc#89]
The following was valid with 0.3.9:

```js
/**
 * @param {string} a      - A for foo.
 * @param {object} opts   - Options.
 * @param {string} opts.b - Option b.
 */
const foo = (a, { b }) => {
  console.log(a, b);
};
foo('a', { b: 'b' });
```

With 0.3.10, prettier now swap the order of the params, which then fails the jsdoc eslint plugin which enforces that the params should be in order.

```bash
─
$ npx eslint ./foo.js

./foo.js
  2:12  warning  Delete `string}·a······-·A·for·foo.⏎·*·@param·{`  prettier/prettier
  4:37  warning  Insert `.⏎·*·@param·{string}·a······-·A·for·foo`  prettier/prettier

✖ 2 problems (0 errors, 2 warnings)
  0 errors and 2 warnings potentially fixable with the `--fix` option.

$ npx prettier -w ./foo.js
foo.js 227ms

$ npx eslint ./foo.js
./foo.js
  2:0  warning  Expected @param names to be "a, a". Got "opts, a"  jsdoc/check-param-names

✖ 1 problem (0 errors, 1 warning)
```

**Comment by hosseinmd** [comment]
Fixed: v0.3.11

**Comment by RunDevelopment** [comment]
@hosseinmd Please reopen. As explained [here](https://github.com/hosseinmd/prettier-plugin-jsdoc/commit/53370180757d32a615e9dbe8f275931b871ca540#r47094217), the current fix isn't guaranteed to work.
